packages/telestion-client-cli/src/lib/generate/widget/index.js

```diff
@@ -0,0 +1,42 @@
+const createWidgetTSX = require('./createWidgetTSX');
+const createWidgetIndexTS = require('./createWidgetIndexTS');
+
+const fs = require('fs');
+const path = require('path');
+const { INSERT_BELOW, INSERT_ABOVE, modifyFile } = require('../../modify-file');
+const logger = require('../lib/logger')('widget-generator');
+
+function generateWidget(argv) {
+	const widgetsFolder = path.join(process.cwd(), 'src', 'widgets');
+	if (
+		!fs.existsSync(widgetsFolder) ||
+		!fs.lstatSync(widgetsFolder).isDirectory()
+	) {
+		logger.error('It looks like your not inside a Telestion PSC Folder.');
+		logger.info("Expected to find folder ./src/widgets, but couldn't.");
+		logger.info('Please re-run this command inside a PSC directory.');
+		process.exit(1);
+	}
+
+	const safeName = argv['name'].toLowerCase().split(' ').join('_');
```

Comment:
```suggestion
	const safeName = argv['name'].toLowerCase().split(' ').join('-');
```
I think, it's reasonable we use `kebab-case` instead of `snake_case` here :wink:

Reply:
Unfortunately, generally speaking, we can't, because `${safe-name}` also gets used as name of the imported constant in line 32 (`import { widget as ${safeName} } from './${safeName}'`).

I could use it for the widget folder, though, if we want that...